src/latency.c

```diff
@@ -555,7 +566,7 @@ void latencySpecificCommandsFillCDF(client *c) {
         if (!cmd->latency_histogram) {
             continue;
         }
-        addReplyBulkCString(c,c->argv[j]->ptr);
+        addReplyBulkSds(c,getFullCommandName(cmd));
         fillCommandCDF(c, cmd->latency_histogram);
         command_with_data++;
```

Comment:
yes, at first I wanna use the COMMAND INFO format, but there is a problem: some parent commands may never be called and generate its latency histogram (i.e. `CONFIG` `CLIENT`), since these commands are virtual (`proc` is NULL), and they are replaced with subcommands in `lookupCommandLogic`, so seems we cannot use the COMMAND INFO directly, maybe need a new format.

I'd like to have your suggestions about the problem above, a parent-sub relation format is clear.

Reply:
maybe i'm missing something, but i think you're wrong.

lookupCommandLogic always returns the sub-command if there is one (the only exception is COMMAND command when argc is 1).

e.g. for both `CONFIG SET` and `CLIENT KILL`: the `INFO commandstats` will show the the sub-comand:
```
cmdstat_config|get:calls=1,usec=34,usec_per_call=34.00,rejected_calls=0,failed_calls=0
cmdstat_info:calls=1,usec=150,usec_per_call=150.00,rejected_calls=0,failed_calls=0
cmdstat_client|kill:calls=1,usec=11,usec_per_call=11.00,rejected_calls=0,failed_calls=1
```
so the latency histogram will be the same.
i.e. if you'll ask for the latency histogram of `CONFIG`, you'll only get the histogram for `config|get` and `config|set`, and the same will happen for CLIENT

Reply:
>lookupCommandLogic always returns the sub-command if there is one (the only exception is COMMAND command when argc is 1).

that's the problem I said, and you give the `INFO commandstats` result, you can see `cmdstat_config` doesn't appear, only the subcommand `cmdstat_config|get`, so how to organize the command (`call=0` and `latency_histogram = NULL`) and its subcommands' structure in `LATENCY HISTOGRAM` reply.

Reply:
the response of LATENCY HISTOGRAM will only contain records of `config|get`, i didn't mean it should include any response for `config`.
what i meant is that if the user asks for `config`, i.e. `LATENCY HISTOGRAM config`, he'll get the results of `config|get` and `config|set`.
that's what COMMAND INFO does too (other than also returning some metadata on CONFIG itself)

Reply:
>that's what COMMAND INFO does too (other than also returning some metadata on CONFIG itself)

did I miss something? I checkout the latest unstable codes, here is the `COMMAND INFO CONFIG`'s reply:
```
1)  1) "config"
    2) (integer) -2
    3) (empty list or set)
    4) (integer) 0
    5) (integer) 0
    6) (integer) 0
    7) 1) @slow
    8) (empty list or set)
    9) (empty list or set)
   10) 1)  1) "config|resetstat"
           2) (integer) 2
           3) 1) admin
              2) noscript
              3) loading
              4) stale
           4) (integer) 0
           5) (integer) 0
           6) (integer) 0
           7) 1) @admin
              2) @slow
              3) @dangerous
           8) (empty list or set)
           9) (empty list or set)
          10) (empty list or set)
...
```
you can see CONFIG itself.

The different between `COMMAND` and `LATENCY` is command is static but latency is dynamic, so subcommands' formant can be same with their parent command, but some parent commands like `CONFIG` would not have any execution and latency info.

Reply:
or you meant the result:
```
127.0.0.1:6379> latency histogram config
1) "config|get"
2) 1) "calls"
   2) (integer) 1
   3) "histogram_usec"
   4) 1) (integer) 33
      2) (integer) 1
3) "config|help"
4) 1) "calls"
   2) (integer) 1
   3) "histogram_usec"
   4) 1) (integer) 33
      2) (integer) 1
```

but it's different with COMMAND INFO config

Reply:
right. what i'm saying is that in the current version of the PR, doing `LATENCY HISTOGRAM config` will yield empty response, and i'm saying that instead it should respond with the latency histograms of all sub-commands.
i.e. you'll get an array with `config|get` and `config|set`, instead of an empty response.

Reply:
yes. that's what i meant. i think it's useful.